src/domain/app/AppYouthProfileRoute.tsx

```diff
@@ -23,21 +22,19 @@ function AppYouthProfileRoute(props: Props) {
   const { t } = useTranslation();
   const [showNotification, setShowNotification] = useState<boolean>(false);
 
-  const [loadProfile, { data, loading }] = useLazyQuery<HasYouthProfile>(
-    HAS_YOUTH_PROFILE,
-    {
-      onError: (error: Error) => {
-        Sentry.captureException(error);
-        setShowNotification(true);
-      },
-    }
-  );
-
-  const isAuthenticated = useSelector(isAuthenticatedSelector);
-
-  useEffect(() => {
-    if (isAuthenticated) loadProfile();
-  }, [isAuthenticated, loadProfile]);
+  const { data, loading } = useQuery<HasYouthProfile>(HAS_YOUTH_PROFILE, {
+    onError: error => {
+      // Ignore permission errors. This decision was made based on the difficulty to add enough checks that slow down Redirecting.
```

Comment:
I know what you mean here because we had a chat on Slack, but my guess is that people without that context might find it difficult to understand what slowing down redirecting means. It's not an easy concept to write in simple terms so I have attempted to give another perspective:

> It's too difficult to block the profile from being loaded before authentication is finished. Instead we allow the UI to optimistically request the profile before it has the authorisation to do so. Because we ignore the subsequent permissions error, the user experience is not affected. Other errors should be shown to the user and be passed to Sentry.

Reply:
That makes much more sense, thanks for writing this. I have a bad habit to write first thing that comes to mind and in most my massage is hard to understand 😅